src/views/settings/Security/Session.vue

```diff
@@ -23,13 +23,13 @@ export default {
               'SECURITY_WATERMARK_ENABLED',
               'SECURITY_SESSION_SHARE',
               'SESSION_EXPIRE_AT_BROWSER_CLOSE',
+              'VIEW_ASSET_ONLINE_SESSION_INFO',
               'SECURITY_MAX_IDLE_TIME',
               'SECURITY_MAX_SESSION_TIME'
             ]
           ]
         ],
-        fieldsMeta: {
-        }
+        fieldsMeta: {}
       }
     }
   }
```

Comment:
There are no apparent discrepancies between the code provided. The structure and functionality appear to be consistent with previous versions of the code. It is recommended that you review any relevant documentation for context on this specific implementation, especially around the Security features like SECURITY_MAX_IDLE_TIME and SECURITY_MAX_SESSION_TIME. If needed, further optimizations would likely focus on ensuring these security limits do not become overly strict based on the current system requirements and user needs.